developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -80,26 +102,91 @@ You can request additional properties by sending a request body with the :code:`
 
 The following properties are supported:
 
-- :code:`{DAV:}getlastmodified`
-- :code:`{DAV:}getetag`
-- :code:`{DAV:}getcontenttype`
-- :code:`{DAV:}resourcetype`
-- :code:`{DAV:}getcontentlength`
-- :code:`{http://owncloud.org/ns}id` The fileid namespaced by the instance id, globally unique
-- :code:`{http://owncloud.org/ns}fileid` The unique id for the file within the instance
-- :code:`{http://owncloud.org/ns}favorite`
-- :code:`{http://owncloud.org/ns}comments-href`
-- :code:`{http://owncloud.org/ns}comments-count`
-- :code:`{http://owncloud.org/ns}comments-unread`
-- :code:`{http://owncloud.org/ns}owner-id` The user id of the owner of a shared file
-- :code:`{http://owncloud.org/ns}owner-display-name` The display name of the owner of a shared file
-- :code:`{http://owncloud.org/ns}share-types`
-- :code:`{http://owncloud.org/ns}checksums`
-- :code:`{http://nextcloud.org/ns}has-preview`
```

Comment:
You need to document the namespaces (because in the table you only use `oc` and `nc`).

Reply:
I don't think the namespaces are relevant for an app developer. No?

Reply:
It is, because the prop lookup always depends on the full namespace uri. The name of the namespace is just there for resolving the uri, but can be anything. When you don't set the correct uri the props using it will be ignored. Be aware that I haven't read the WebDAV spec, but this is how it works afaik.

Reply:
Ah, I get what you mean, you're right this is needed :+1: